### ISSUE TITLE
Tapping "Show more actions" just hides the actions bar

### ISSUE THREAD

**Issue by WofWca** [laurent22/joplin#10867]
### Operating system

Android

### Joplin version

3.1.1

### Current behaviour

1. Open a note
2. Tap "edit". The actions bar will appear (with "search", "bold", "italics", etc)
4. Tap the three dots in the actions bar (i.e. "Show more actions")

The actions bar just disappears until you leave editing mode.

### Expected behaviour

The bottom bar expands and shows more actions

***

I believe this started happening only after the update from 3.0.9 to 3.1.1.
I have tried disabling plugin support: did not help.

**Comment by bsavant** [comment]
This is still not working in iOS 13.1.3

**Comment by venetin** [comment]
This is still brokren on 3.1.2 Android!

**Comment by dabreese00** [comment]
I also still have this same symptom on version 3.1.2 on Android. I can reproduce it 100% of the time. Let me know if I can help.

**Comment by laurent22** [comment]
For me it is solved as of 3.1.3. If it's still happening of one of you please open a new issue and specify the needed information
